modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -11,6 +11,11 @@ The OVN-Kubernetes Container Network Interface (CNI) cluster network provider ha
 The default value, `cluster`, is supported for both parameters.
 This limitation can affect you when you add a service of type `LoadBalancer`, `NodePort`, or add a service with an external IP.
 
+[NOTE]
+====
+Openshift Installer provisioned infrastructure (IPI) installation on Azure cloud, the installer create Loadbalancer service with `ExternalTrafficPolicy` pointing to local, however the load is still distributed to the available backends until the fix is available, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1772847[BZ#1903408].
```

Comment:
It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.  

The link URL has one BZ number, and the link text has a different number.  I think you want the latter: <https://bugzilla.redhat.com/show_bug.cgi?id=1903408>.  

Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing text?

Reply:
Hello @Miciah 

Thank you for the comment.

> It shouldn't matter whether it's IPI or UPI, and the issue would apply to other platforms as well as Azure.

Correct the lines a bit to include just an ingress service. 
I think, we only create such LB service type in Azure, I'm not seeing ETP set in UPI. 
When I create LB type svc using oc expose for any random service, it set ETP to cluster which is correct. 

> The link URL has one BZ number, and the link text has a different number. I think you want the latter: https://bugzilla.redhat.com/show_bug.cgi?id=1903408.

Corrected. 

> Is the intent here to add a BZ citation, or is there additional information that isn't covered by the existing text?

Yes, the OVN limitation is confusing when it comes to the Ingress service that we create as part of the installation with respect to ETP. This change will just give more clarity on it.

Reply:
> I think, we only create such LB service type in Azure, I'm not seeing ETP set in UPI.

Whether the cluster uses IPI or UPI, the resulting cluster is supposed to be the same.  Anyway, the ingress operator looks at the platform type in the infrastructure config (which you can check in `oc get infrastructures.config/cluster`), so UPI or IPI shouldn't make a difference.  

We default to "LoadBalancerService" for all cloud platforms; see <https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L84-L102> and <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/pkg/operator/controller/ingress/controller.go#L359-L360>.  We set `externalTrafficPolicy` on LoadBalancer services on all platforms except for IBM Cloud and Power VS; see <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/assets/router/service-cloud.yaml#L14-L16> and <https://github.com/openshift/cluster-ingress-operator/blob/5040f65551851b3ee284f0803bfdd1c64631c4c6/pkg/operator/controller/ingress/load_balancer_service.go#L401-L406>.  

> Yes, the OVN limitation is confusing when it comes to the Ingress service that we create as part of the installation with respect to ETP. This change will just give more clarity on it.

Is the problematic behavior specific to Azure?  If not, then I'm concerned that adding a note saying that the problem existed on Azure would imply that it didn't exist on other platforms and would mislead users.

Reply:
Thank you @Miciah, I have applied correction accordingly.